features/managing_number_of_items_on_channel.feature

```diff
@@ -0,0 +1,22 @@
+@managing_channels
+Feature: Managing number of items on channel
```

Comment:
It's not detailed enough IMHO. I'd specify which items eg. `Managing number of items to synchronize on channel`